### PR TITLE
bind port network connection details (SYS_BIND)

### PR DESCRIPTION
**Pull request by yasin-cs-ko-ak**
This PR Contains bind port connection for summarised data
- Using SYS_BIND to get bind port
- `sin_port` and `sin_addr` used for bind port and bind address respectively.

### CODE REVIEW

**Review by vishnusomank — APPROVED** [review]

File: src/protobuf/v1/observability/observability.proto [hunk 1]
```diff
@@ -29,8 +29,9 @@ message Response{
     repeated SysProcFileSummaryData FileData = 7;
     repeated SysNwSummaryData InNwData = 8;
     repeated SysNwSummaryData OutNwData = 9;
-    repeated CiliumSummData IngressData = 10;
-    repeated CiliumSummData EgressData = 11;
+    repeated SysNwSummaryData BindNwData = 10;
```
Comment (seswarrajan) [inline src/protobuf/v1/observability/observability.proto:32]: Just a small suggestion. Whenever adding a new entry to protobuf struct, make sure to add it at the end.

**Review by seswarrajan — APPROVED** [review]
LGTM.

File: src/protobuf/v1/observability/observability.proto (hunk 1 above)
Comment (yasin-cs-ko-ak) [reply src/protobuf/v1/observability/observability.proto:32]: Changes are made in another PR. Kindly review it. Because this PR has unresolved conflicts.

**Comment by yasin-cs-ko-ak** [comment]
Closing this PR because unsolvable conflict with protobuf

**Pull request closed without merging**